liberum_cli/src/main.rs

```diff
@@ -140,11 +143,28 @@ struct PublishFile {
     path: PathBuf,
 }
 
+#[derive(Parser)]
+struct GetPublishedObjects {
+    #[arg()]
+    node_name: String,
+}
+
 #[derive(Tabled)]
 struct NodeInfoRow {
     pub name: String,
     pub is_running: bool,
-    pub first_address: String,
+    pub first_cfg_address: String,
```

Comment:
We definitely need an option to list all cfg addresses and all listen addresses. It is fine to print only one in the list-nodes